core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-category-field-search-list/dot-category-field-search-list.component.scss

```diff
@@ -0,0 +1,11 @@
+.category-field__search-list {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    overflow: hidden;
+}
+
+::ng-deep .dotTable.p-datatable > .p-datatable-wrapper {
```

Comment:
add the host selector, to don't be general.

Reply:
fixed